app.js

```diff
@@ -0,0 +1,28 @@
+const {getCategories} = require("./controllers/controllers")
+const express = require("express")
+const app = express();
+app.use(express.json())
+
+app.get("/api/categories", getCategories);
+
+
+app.all("/*", (req, res) => {
+    res.status(404).send({ msg: "Route not found" });
+  });
+
+app.use((err, req, res, next) => {
```

Comment:
This custom error handler doesn't need to be here yet.
Best to take it out for now and re-introduce it later on when it is needed - with accompanying tests.